tough/src/schema/mod.rs

```diff
@@ -171,12 +230,43 @@ pub struct Snapshot {
     pub _extra: HashMap<String, Value>,
 }
 
+/// Represents a metadata file in a `snapshot.json` file.
+/// TUF 4.4: METAFILES is an object whose format is the following:
+/// ```text
+///  { METAPATH : {
+///        "version" : VERSION,
+///        ("length" : LENGTH, |
+///         "hashes" : HASHES) }
+///    , ...
+///  }
+/// ```
+/// e.g.
+/// ```json
+///    "project1.json": {
+///     "version": 1,
+///     "hashes": {
+///      "sha256": "f592d072e1193688a686267e8e10d7257b4ebfcf28133350dae88362d82a0c8a"
+///     }
+///    },
+/// ```
 #[derive(Debug, Clone, Deserialize, Serialize, PartialEq)]
 pub struct SnapshotMeta {
+    /// TUF 4.4: LENGTH is the integer length in bytes of the metadata file at METAPATH. It is
```

Comment:
```suggestion
    /// LENGTH is the integer length in bytes of the metadata file at METAPATH. It is
```
Don't think we need the prefixes again?  They weren't repeated in the last section.